cmd/generator/main.go

```diff
@@ -87,6 +91,20 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-nps",
 		}
 
+		type iconURL struct {
+			URL    string
+			Inline bool
+		}
+
+		iconURLs := map[string]iconURL{
+			"mattermost-plugin-aws-SNS":        iconURL{"https://unpkg.com/simple-icons@latest/icons/amazonaws.svg", true},
```

Comment:
Should we change this to point to `integrations.mattermost.com`?

Reply:
This is just for the `generator` script, which pulls down these assets and embeds the base64 data into the `plugins.json`. The server won't ever see or use these URLs.